python/ray/util/sgd/v2/worker_group.py

```diff
@@ -70,17 +70,21 @@ def __init__(self,
         self.workers = []
         self.start()
 
+    def _create_worker(self):
+        remote_cls = ray.remote(
+            num_cpus=self.num_cpus_per_worker,
+            num_gpus=self.num_gpus_per_worker)(BaseWorker)
```

Comment:
we probably don't want to call `ray.remote ` too many times; I think you should just do it once upon initialization of the worker group class

Reply:
Good point, fixed!